### PR TITLE
New "View on GitHub" option, more user friendly

### PR DESCRIPTION
**Pull request by NokiaGuy**
More user-friendly:
--
More user-friendly dialog's for new users, since RLW is pretty popular. Not everyone is completely use to ROBLOX's terms. 

---

Change "Roblox Studio" to "Develop on Roblox (Roblox Studio)":
--
ROBLOX's community throws around the term "develop" more than "ROBLOX Studio" sadly :C. Newcomers will know what's what.

---

Change "Legacy Mode" to "Basic Mode":
--
Legacy mode is an odd name when it is just as good as the browser mode, if not better. Much quicker.

### CODE REVIEW

**Comment by GLolol** [comment]
The only meaningful change I see here is the GitHub link. I don't see how anything else is "more user friendly".

File: rlw.sh [hunk 1]
```diff
@@ -98,7 +98,7 @@ rwinetricks () {
 roblox-install () {
 	printf '%b\n' " > begin roblox-install ()\n---"
 	if [[ ! -d "$WINEPREFIX/drive_c" ]]; then
-		spawndialog question 'A working Roblox wineprefix was not found.\nWould you like to install one?'
+		spawndialog question 'Roblox Linux Wrapper doesnt seem to be installed.\nWould you like to install it?'
 		if [[ $? = "0" ]]; then
```
Comment (GLolol) [inline rlw.sh]: Installation of the script in this case is not the same as creating the wineprefix. It's better to be specific in these messages.

**Comment by NokiaGuy** [comment]
Going to add fixes, sorry about that!

File: rlw.sh [hunk 2]
```diff
@@ -112,8 +112,8 @@ roblox-install () {
 			wget -N -r --no-parent -Aexe http://download.cdn.mozilla.net/pub/mozilla.org/firefox/releases/latest-esr/win32/en-US/ -nd -P /tmp/Firefox-Setup/
 			WINEDLLOVERRIDES="winebrowser.exe,winemenubuilder.exe=" rwine /tmp/Firefox-Setup/*.exe /SD | zenity \
 				--window-icon="$RBXICON" \
-				--title='Installing Mozilla Firefox' \
-				--text='Installing Mozilla Firefox Browser ...' \
+				--title='Installing (Windows) Mozilla Firefox' \
+				--text='This is so you can launch games.\nDo not worry, this will not mess up your current Linux FireFox installation.' \
```
Comment (GLolol) [inline rlw.sh]: This doesn't sound professional. The old language is concise enough, and Firefox is spelt with only the first F capitalized.

File: rlw.sh [hunk 3]
```diff
@@ -130,6 +130,7 @@ roblox-install () {
 
 playerwrapper () {
 	printf '%b\n' " > begin playerwrapper ()\n---"
+	spawndialog warning "Make sure you've logged in to either Studio or Firefox.\nWithout being logged in, this will make you join as a guest."
```
Comment (GLolol) [inline rlw.sh]: We have enough popup dialogs already... Can you at least add some check so this doesn't show up every time?

Comment (alfonsojon) [inline rlw.sh]: In order to determine if the user is logged in via Firefox or the studio, we would need to do some highly advanced cookie detection which extends far beyond the scope of this project.

**Pull request closed without merging**